antora-playbook.yml

```diff
@@ -3,6 +3,8 @@ site:
   url: https://red-hat-storage.github.io/ocs-sop
   start_page: sop::index.adoc
   robots: allow
+  keys:
+    google_analytics: 'G-LGCEEZGN54'
```

Comment:
You copied the ID from ocs-training... please use your own ID `G-PDMKGVZNGE`

Reply:
Just waking up... fixed.